src/components/TaskCard/AddTaskCard.jsx

```diff
@@ -31,6 +31,11 @@ const AddTaskCard = () => {
     );
     setInputText(null);
   };
+
+  const cancelAction = () => {
+    dispatch(createButton());
+  };
```

Comment:
Why `cancelAction` is dispatching `createButton` ?

Reply:
Add Task Card should be hidden on cancel action and "create" button should be enabled . May be I used wrong name for this action . renamed "createButton" action to "toggleCreateButtonVisibility" . Thank you